### DISCUSSION TITLE
[4.1] - How to change default way to store refreshToken?

### DISCUSSION THREAD

**Discussion by rhadamez** [adonisjs/core#2008] [Help] [answered]
Hey you!

By default, refreshToken is stored at table "token" with the "await auth.withRefreshToken().attempt(email, password)". My question is: how could I change this default behavior to store in another table?

Another question is: how store the refreshToken in redisdb? I would like to use the "auth.withRefreshToken().attempt(email, password)" for it.

Well, a lot of things.

![refreshToken](https://user-images.githubusercontent.com/18365154/101183394-39bcc480-362e-11eb-888a-2ad1b33f9534.png)

**Comment by thetutlage** [comment, marked as answer]
There is a model for tokens and you can change the table name on it. Regarding redis, v4 doesn't allow that until you want to create your own provider for persistence.
